modules/ROOT/pages/cloudhub-architecture.adoc

```diff
@@ -20,33 +20,31 @@ Manage CloudHub with the Anypoint Runtime Manager console in Anypoint Platform.
 See xref:deployment-strategies.adoc[Deployment Options] for a better understanding of the different possible deployment scenarios, including on-premises and Anypoint Platform Private Cloud Edition (Anypoint PCE).
 ====
 
-== CloudHub Architecture
-
 To understand CloudHub’s approach to security and availability, it’s important to understand the architecture behind CloudHub.  It includes two major components--Anypoint platform services, and the worker cloud. These two components and the Runtime Manager console through which you access them work together to run your integration applications.
 
 image::architecture-diagram.png[]
 
 [cols="10a,45a,45a"]
 |===
-|  |<<Integration Applications>> |These are applications that you create and deploy to CloudHub to perform integration logic for your business.
-|image:blue-1.png[] |<<Runtime Management Console>> |The Runtime Manager console is the face of CloudHub, allowing you to deploy and monitor integrations, and configure your account, among other things.
+|  |<<Integration Applications>> |The applications that you create and deploy to CloudHub to perform integration logic for your business
```

Comment:
Not a big deal, but I would take "The" out of these descriptions to stay parallel and avoid The The The

Reply:
agreed